### PR TITLE
TFAC-1118 Getting SFAC Activity State

### PR DESCRIPTION
**Pull request by jedtan**

### CODE REVIEW

File: graphql/data/schema.js [hunk 1]
```diff
@@ -765,10 +766,16 @@ const userType = new GraphQLObjectType({
       resolve: (user, _, context) =>
         getShouldShowSfacExtensionPrompt(context.user, user),
     },
+    sfacActivityState: {
+      type: new GraphQLNonNull(GraphQLString),
```
Comment (kmjennison) [inline graphql/data/schema.js:770]: Let's make this an enum so the possible options are discoverable.

File: graphql/data/schema.js [hunk 2]
```diff
@@ -765,10 +766,16 @@ const userType = new GraphQLObjectType({
...
+      description: 'what mode in which to show SFAC searches UI',
+      resolve: (user, _, context) => getSfacActivityState(context.user, user),
+    },
     yahooPaidSearchRewardOptIn: {
       type: new GraphQLNonNull(GraphQLBoolean),
-      description:
-        'whether or not the user has opted into searching for extra impact',
+      description: 'status ',
+      resolve: (user, _, context) =>
+        getShouldShowSfacExtensionPrompt(context.user, user),
```
Comment (kmjennison) [inline graphql/data/schema.js:778]: Are these changes to yahooPaidSearchRewardOptIn intentional? If so, could you help me understand what they're doing?

**Review by kmjennison — CHANGES_REQUESTED** [review]

File: graphql/data/schema.js (hunk 2 above)
Comment (jedtan) [reply graphql/data/schema.js:778]: Err.... no idea! My bad!

File: web/data/schema.graphql [hunk 3]
```diff
@@ -1618,7 +1618,10 @@ type User implements Node {
   """whether to show the SFAC extension prompt"""
   showSfacExtensionPrompt: Boolean!
 
-  """whether or not the user has opted into searching for extra impact"""
+  """what mode in which to show SFAC searches UI"""
+  sfacActivityState: String!
```
Comment (kmjennison) [inline web/data/schema.graphql:1622]: Need to regenerate schema.graphql.

**Review by kmjennison — CHANGES_REQUESTED** [review]
Looks good! Once you build schema.graphql, feel free to merge.

**Pull request merged**